ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -34,6 +35,30 @@ Create Pipeline Server
     Wait Until Generic Modal Disappears
     Wait Until Project Is Open    project_title=${project_title}
     ...    timeout-pre-spinner=5s    timeout-spinner=60s
+    # Check if there is any CA Bundle
+    # I did not find a better way to check if multi-line string is empty or "NONE"
+    ${length}=    Get Length    "${aws_s3_ca_bundle}"
+    IF    ${length} <= 6
+        ${aws_s3_ca_bundle}=    Get Variable Value    ${S3.AWS_CA_BUNDLE}    ${NONE}
+        ${length}=    Get Length    "${aws_s3_ca_bundle}"
+    END
+    IF    ${length} > 6
+        Patch Pipeline Server With CA Bundle
+        ...    project_title=${project_title}    aws_s3_ca_bundle=${aws_s3_ca_bundle}
+    END
+
+Patch Pipeline Server With CA Bundle
+    [Documentation]    Patch the underlying dspa resource with CA Bundle
```

Comment:
Please add in the `Documentation` section what we should do.
Example:
```
Add S3.AWS_CA_BUNDLE with the certificate text into the test-variables.yml file in order to have the server patched
```

Reply:
I have added
```
The CA Bundle can be obtained from S3.AWS_CA_BUNDLE in test-variables.yml
```